test/state/host.cpp

```diff
@@ -88,10 +88,17 @@ bytes_view extcode(bytes_view code) noexcept
 }
 
 /// Check if an existing account is the "create collision"
-/// as defined in the [EIP-684](https://eips.ethereum.org/EIPS/eip-684).
+/// as defined in the [EIP-7610](https://eips.ethereum.org/EIPS/eip-7610).
 [[nodiscard]] bool is_create_collision(const Account& acc) noexcept
 {
-    return acc.nonce != 0 || !acc.code.empty();
+    if (acc.nonce != 0 || !acc.code.empty())
+        return true;
+
+    // The storage may have access list entries so look for non-zero values.
```

Comment:
it was not very clear at first, I think you meant
```suggestion
    // acc.storage may have entries from access list, even if account storage is empty. Check for non-zero current values.
```